### PR TITLE
DOCS-90: Update TrueNAS guide to use correct active/standby controlle…

### PR DESCRIPTION
**Pull request by Mrt134**
…r terminology.

- Add new substitutions to cover a few edge cases for the standard terms.
- Grep through all the .rst files and change every found instance of active, passive, standby controller to the new substitutions.
- TrueNAS HTML build test: no issues.

### CODE REVIEW

File: userguide/snippets/ui_login.rst [hunk 1]
```diff
@@ -50,10 +50,10 @@ about the active |ctrlr-term| is displayed on this screen. Log in with:
 On the first login, the EULA found in :ref:`Appendix A` is displayed,
 along with a box where the license key for the %brand% array can be
 pasted. Read the EULA and paste in the license key. High Availability
-(HA) systems must have both active and standby |ctrlrs-term| booted
-before the license key for the HA %brand% system can be entered. The key
-is entered on the active |ctrlr-term|. Click :guilabel:`OK` to save the
-license key and access the |web-ui|.
+(HA) systems must have both |ctrlrs-term-both| booted before the license
```
Comment (wblock) [inline userguide/snippets/ui_login.rst]: "Booted" implies they are starting up, but really they both have to be fully up.
```suggestion
(HA) systems must have both |ctrlrs-term-both| running before the license
```

File: userguide/conf.py [hunk 2]
```diff
@@ -87,6 +87,10 @@
 .. |ctrlr-term-1-2|       replace:: TrueNAS controller 1/2
 .. |ctrlr-term-active|    replace:: active TrueNAS controller
 .. |ctrlr-term-standby|   replace:: standby TrueNAS controller
+.. |ctrlrs-term-both|     replace:: active and standby TrueNAS controllers
+.. |active|               replace:: active
+.. |standby|              replace:: standby
+.. |actv-stndby|          replace:: active/standby
```
Comment (wblock) [inline userguide/conf.py]: Abbreviating this one when the others are not abbreviated is inconsistent and will probably lead to errors.
